pkg/controller/schedulingmanager/controller.go

```diff
@@ -164,7 +163,7 @@ func (c *SchedulerController) reconcile(qualifiedName util.QualifiedName) util.R
 	}
 
 	glog.Infof("Start plugin with kind %s for scheduling type %s", templateKind, kind)
-	err = c.scheduler[kind].StartPlugin(templateKind, &apiResource, c.stopChan)
+	err = c.scheduler[kind].StartPlugin(typeConfig, c.stopChan)
```

Comment:
not necessarily relevant to this PR; however came across this thought while looking at this code wrt to your changes in https://github.com/kubernetes-sigs/federation-v2/pull/427.
I think its possible to create multiple `TypeConfigs` for the same target kind, (perhaps a possible mistake by whoever creates them). For example a `TypeConfig` was created with name `deployments.apps` for `deployment` as target as part of the control plane deployment automation. Later an admin creates a `TypeConfig` with name `deployments` for same target. Multiple schedulers would be created here, (and possibly multiple sync controllers for same target). Does it make sense to handle it in some way, or this seems to be a remote possibility, or .... ?

Reply:
Good catch.  I've internalized the expectation that type configs are named `<target plural name>.<target group name>`, such that there will never be more than one type config for a given target for a given federation deployment.  When I checked that assumption, though, I realized that no validation exists to preclude this possibility.  I've added #433 to track resolution.

Reply:
Ack.